lib/pitchfork/configurator.rb

```diff
@@ -79,6 +79,7 @@ class Configurator
       :client_body_buffer_size => Pitchfork::Const::MAX_BODY,
       :before_service_worker_ready => nil,
       :before_service_worker_exit => nil,
+      :setpgid => false,
```

Comment:
We may want to use a different key/method name here.

Reply:
Yeah, I don't know. At least it's pretty clear which syscall is involved. But people may think they're meant to give a numeric ID or something.

Honestly I think it's fine, but if you have a better idea let me know.

Reply:
I think it's fine, tbh, but wanted to mention it just in case.